services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_mini.jsx

```diff
@@ -92,11 +123,30 @@ export default class UnitTemplateMini extends React.Component {
     }
   }
 
+  setShowToolTip = () => {
+    this.setState({ showTooltip: true });
+  }
+
+  setHideToolTip = () => {
+    this.setState({ showTooltip: false });
+  }
+
   render() {
+    const { showTooltip } = this.state;
     return (
-      <div className='unit-template-mini' onClick={this.onClickAction}>
+      <div className='unit-template-mini' onClick={this.onClickAction} onMouseEnter={this.setShowToolTip} onMouseLeave={this.setHideToolTip}>
```

Comment:
Why do we need to do this? Couldn't the whole element just be the tooltip trigger? Example here: https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/ell_filters.tsx#L199-L200.

Reply:
oh, I didn't know that was possible-- I'll revert these changes